regression/apps/fdatasync/fdatasync.c

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MPL-2.0
+
+#include <stdio.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <string.h>
+#include <stdlib.h>
+
+void test_fdatasync_on_fs(const char *directory)
+{
+	char filepath[256];
+	snprintf(filepath, sizeof(filepath), "%s/test_fdatasync.txt",
+		 directory);
+
+	int fd =
+		open(filepath, O_WRONLY | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR);
```

Comment:
Combine the two lines into one

Reply:
I tried it before. However, it's strange that our format tool enforces it to two separate lines.